tests/conftest.py

```diff
@@ -377,3 +380,14 @@ async def iframe_id(websocket, context_id: str, html_iframe_same_origin, html):
     await goto_url(websocket, iframe_id, html("<h1>FRAME</h1>"))
 
     return iframe_id
+
+
+@pytest_asyncio.fixture
+async def user_context(websocket):
```

Comment:
```suggestion
async def create_user_context(websocket):
```

Reply:
The fixture is supposed to be used as a parameter in the test, and will already provide the result.

Reply:
There is a fixture called create_user_context in WPT and I think to makes sense to match the style, also `create_context` would be a similar feature.

Reply:
Currently, the `create_context` returns a function, which can be called in test. Al long as this fixture follows the naming pattern used in this file, I suggest moving this out of scope of this PR.

Reply:
well, I am not sure there is a pattern in this file :) there is `context_id` fixture, should this one be called `user_context_id`?

Reply:
`user_context_id` SGTM. Done.